File: scripts/hash_api_key.py
import bcrypt

api_key = b'ak_da955087b123f8edee27231d122b2a9f50e98d847dc68646b9214af7153b85bc'

# Generate hash with bcrypt (salt rounds = 10)
hashed = bcrypt.hashpw(api_key, bcrypt.gensalt(rounds=10))

print('\n========================================')
print('  🔒 API Key Hash Generated')
print('========================================\n')
print('API_KEY_HASH:')
print(hashed.decode('utf-8'))
print('\n========================================')
print('📋 Copia esto a Railway:')
print('========================================\n')
print(f'API_KEY_HASH={hashed.decode("utf-8")}')
print('\n========================================\n')

File: scripts/archive/legacy-key-generation/hash-api-key.js
const bcrypt = require('bcryptjs');

const apiKey = 'ak_da955087b123f8edee27231d122b2a9f50e98d847dc68646b9214af7153b85bc';

bcrypt.hash(apiKey, 10).then(hash => {
  console.log('\n========================================');
  console.log('  🔒 API Key Hash Generated');
  console.log('========================================\n');
  console.log('API_KEY_HASH:');
  console.log(hash);
  console.log('\n========================================\n');
}).catch(console.error);
